bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -24,13 +24,13 @@ metadata:
     containerImage: icr.io/cpopen/common-service-operator:latest
     createdAt: "2020-10-19T21:38:33Z"
     description: The IBM Common Service Operator is used to deploy IBM Common Services
-    nss.operator.ibm.com/managed-operators: cloud-native-postgresql,ibm-bts-operator
+    nss.operator.ibm.com/managed-operators: cloud-native-postgresql, ibm-bts-operator
```

Comment:
Additional space in string `cloud-native-postgresql,ibm-bts-operator` will cause NamespaceScope Operator failure.

Let's change [original file](https://github.com/IBM/ibm-common-service-operator/blob/release-ltsr/config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml#L10) in `config` file as well